app/gui/src/project-view/components/ExtendedMenu.vue

```diff
@@ -17,6 +17,7 @@ const open = defineModel<boolean>('open', { default: false })
     <template #menu>
       <ActionMenu
         :actions="[
+          'graph.navigateUp',
```

Comment:
This might cause a conflict later, and might need to be moved to `GraphEditor.vue` once https://github.com/enso-org/enso/pull/12601 lands.

https://github.com/enso-org/enso/pull/12601/files#diff-d8f9b89e80abaa1e78ebeb4529995f0ae8380fea71013d4c460dc1286809ea26R664-R674